app/models/user.rb

```diff
@@ -35,4 +36,14 @@ def age
     bornDate = Time.zone.parse(birthday.to_s)
     ((bornDate.to_date...currentDate.to_date).count - 1) / 365.0
   end
+
+  def jwt_payload
+    {
+      "context": {
+        "id": id,
+        "name": name,
+        "role": role,
+      },
+    }
+  end
```

Comment:
This is something not related to model and should be moved out

Reply:
According to the tutorial I followed, if I wanted to add more things to the payload, I needed to declare the method in the user model and return a hash with the new values I wanted to add